cves/kernel/CVE-2017-15126.yml

```diff
@@ -377,8 +383,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    The fix puts and if around the code it was doing and a else to do something
+    else in the situation that was causing the error.
```

Comment:
I think it may be more effective to say that a conditional was added to handle the error. This was kinda hard to read